binderhub/static/js/index.js

```diff
@@ -10,8 +10,17 @@
   pushing -> built
   pushing -> failed
 */
+import * as Terminal from 'xterm';
+import 'xterm/lib/xterm.css';
+import 'bootstrap';
 
-"use strict";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap-theme.min.css';
+import '../index.css';
+
+
+// FIXME: Can not seem to import this addon from npm
+import {fit} from './vendor/xterm/addons/fit';
```

Comment:
These docs have a different approach to loading add-ons:

https://xtermjs.org/docs/api/addons/fit/#undefined

https://github.com/xtermjs/xterm.js#addons

Reply:
Yep, except they don't seem to work with webpack. https://github.com/xtermjs/xterm.js/issues/1018 has more details.

Reply:
Might as well toss the issue into the comment too. Yay, JS fun.

Reply:
Is the use of xterm here just for handling of stdout?

If you're less worried about full terminal support and just want to render basic color, deal with carriage returns, etc. there are packages available, namely:

* https://github.com/lgeiger/escape-carriage
* anser, ansi-to-react

Reply:
yeah, it's for handling the output of docker build. Unfortunately it often has ansi control characters in it, since it is just arbitrary programs outputting to stdout. Will take a look at the other packages!

Reply:
/cc @lgeiger

Reply:
@willingc I added the issue link as a comment!